### PR TITLE
Fix 25936

### PR DESCRIPTION
**Pull request by divyajyothi5321**
### Fixed Issues (if relevant)

1. magento/magento2#25936: Regular Price Label Alignment Issues in Frontend

### Manual testing scenarios (*)

1. Admin->Catalog->Products->Add Product
2. Add Price ex:$20
3. Click on Advanced Pricing and add regular price ex:$10
4. Save the product
5. Open product in frontend 
6. you can see the issue is fixed with this code as shown in the below screenshot.
regularprice-expectedresult

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by kanthy-i95dev — APPROVED** [review]

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25989.instances.magento-community.engineering).
Admin access: [https://pr-25989.instances.magento-community.engineering/admin](https://pr-25989.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by ptylek — CHANGES_REQUESTED** [review]
@divyajyothi5321 Thanks for your contribution. Your solution breaks SKU into two lines, see the photo:
image

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less [hunk 1]
```diff
@@ -648,6 +648,20 @@
...
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
+    .product-info-main {
+        .product-info-price {
+            .price-box {
+                width: 1px;
+            }
+        }
+    }
+}
+
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:664]: Doesn't make sense. Looks like this:
Screenshot 2019-12-12 at 20 31 02

Comment (ptylek) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:664]: And I guess you want to achieve this:
image

**Review by ptylek — CHANGES_REQUESTED** [review]
Your changes break line on price box and sku. Screenshot was taken on ipad.
Screenshot 2019-12-12 at 20 30 29

**Comment by ptylek** [comment]
@divyajyothi5321, I am closing this PR now due to inactivity. Please reopen and update if you wish to continue. Thank you for collaboration

**Pull request closed without merging**